packages/@sanity/desk-tool/src/panes/document/timeline/__workshop__/TimelineItemStory.tsx

```diff
@@ -0,0 +1,95 @@
+import {Card, Container, Flex, LayerProvider, Menu} from '@sanity/ui'
```

Comment:
What's our stance on adding things to the workshop and push them to next? Should we keep it as a dev tool or keep it as well and push it to next? 🤔